src/resources/employees.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+
+const router = express.Router();
+const fs = require('fs');
+const employees = require('../data/employees.json');
+
+module.exports = router;
+
+router.post('/createEmployee', (req, res) => {
+  const employeeName = req.body.first_name;
+  const employeeLastName = req.body.last_name;
+  const employeeEmail = req.body.email;
+  const employeePassword = req.body.password;
+  const employeeTask = req.body.task;
+  const employeeProject = req.body.projects;
+  const newEmployee = {
+    id: new Date().getTime().toString().substring(6),
+    first_name: employeeName,
+    last_name: employeeLastName,
+    email: employeeEmail,
+    task: employeeTask,
+    password: employeePassword,
+    projects: employeeProject,
+  };
+  employees.push(newEmployee);
+  fs.writeFile('src/data/employees.json', JSON.stringify(employees, null, 2), (err) => {
+    if (err) {
+      res.send('Cannot create new employee');
+    } else {
+      res.send('New employee created successfully');
+    }
+  });
+});
+
+router.put('/editEmployee', (req, res) => {
+  const employeeId = req.body.id;
+  const employeeName = req.body.first_name;
+  const employeeLastName = req.body.last_name;
+  const employeeEmail = req.body.email;
+  const employeePassword = req.body.password;
+  const employeeTask = req.body.task;
+  const employeeProject = req.body.projects;
+  const employeeToBeEdited = {
+    id: employeeId,
+    first_name: employeeName,
+    last_name: employeeLastName,
+    email: employeeEmail,
+    task: employeeTask,
+    password: employeePassword,
+    projects: employeeProject,
+  };
+  let newEmployeesList = employees;
+  newEmployeesList = newEmployeesList.map((employee) => {
```

Comment:
The assignment newEmployeesList = employees is not necessary here, we can change this piece of code to,
```suggestion
  const newEmployeesList = employees.map((employee) => {
```